core/tests/karma-globals.js

```diff
@@ -16,4 +16,15 @@
  * @author sll@google.com (Sean Lip)
  */
 
-var GLOBALS = {};
\ No newline at end of file
+var GLOBALS = {};
+
+GLOBALS.OVERWRITE_TRANSLATOR_PROVIDER =  function($provide, $translateProvider) {
```

Comment:
nit: remove extra space after '='

Can you add a comment above this line explaining what this does?